.github/workflows/build-and-test.yml

```diff
@@ -21,6 +21,8 @@ jobs:
     with:
       machine-types: '["windows-2022"]'
       timeout-minutes: 60
+      blame-hang-timeout: 1800000
+      enable-test-diagnostic-mode: 'true'
```

Comment:
@sarahelsaig isn't this just a temporary change for investigating the issue?

Reply:
It's temporary, but should stay for now: https://github.com/OrchardCMS/OrchardCore.Commerce/pull/526#discussion_r1987068238

> Also, I'd like to keep in the blame hang timeout in the workflow for the time being, so we get a followup if there are any other freezy points.